SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -750,6 +804,20 @@ void CTurbSSTSolver::BC_Outlet(CGeometry *geometry, CSolver **solver_container,
       conv_numerics->SetGridVel(geometry->nodes->GetGridVel(iPoint),
                                 geometry->nodes->GetGridVel(iPoint));
 
+      if(bounded_scalar){
+        /*--- Obtain flow velocity vector at inlet boundary node ---*/
+        su2double Velocity_Outlet[MAXNDIM] = {0.0}, density_outlet;
+        for (auto iDim = 0u; iDim < nDim; iDim++) Velocity_Outlet[iDim] = solver_container[FLOW_SOL]->GetNodes()->GetVelocity(iPoint, iDim);
+        
```

Comment:
Too much duplication and I had comments here regarding the correctness of this flux that were marked resolved, but I don't see any difference in the code, or reply to the comment.